chrome/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "chrome",
+  "version": "0.4.0",
+  "description": "Click to report an incompatible site at webcompat.com, and optionally upload a screenshot.",
+  "author": "Mike Taylor",
+  "main": "background.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "eslint": "eslint -c package.json --fix background.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/webcompat/webcompat-reporter-extensions.git"
+  },
+  "bugs": {
+    "url": "https://github.com/webcompat/webcompat-reporter-extensions/issues"
+  },
+  "homepage": "https://github.com/webcompat/webcompat-reporter-extensions#readme",
+  "devDependencies": {
+    "jpm": "^1.1.4",
+    "eslint": "^1.8.0",
```

Comment:
Thanks!

(and thanks for being patient... traveling at the moment for a conference)